### PR TITLE
Minor coding standards related stuff

### PR DESCRIPTION
**Pull request by Reggino**
- Added .editorconfig for EditorConfig support
- Add yarn.lock to .gitignore since it's widely used
- Minor coding standards issue

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+indent_style = space
+indent_size = 2
```
Comment (n1k0) [inline .editorconfig]: Nit: Please ass a trailing CR to avoid diff issues.

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,4 @@ node_modules
 build
 dist
 lib
-
+yarn.lock
```
Comment (n1k0) [inline .gitignore]: Nit: Please ass a trailing CR to avoid diff issues.

File: .editorconfig (hunk 1 above)
Comment (Reggino) [inline .editorconfig]: Thanks: done!

**Comment by n1k0** [comment]
Thank you :+1:

**Pull request merged**